TranslationStudioAutomation/Sdl.StudioAutomation.Samples.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31005.135
+# Visual Studio Version 17

```

Comment:
Please keep this at Vs2019 for the moment

Reply:
Done